<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Foods</class>
 <widget class="QWidget" name="Foods">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="bill_master_resources.qrc">:/icons/icons/foods_small.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Foods</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="addFood">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Add Foods and  Beverages</string>
       </property>
       <property name="icon">
        <iconset resource="bill_master_resources.qrc">
         <normaloff>:/icons/icons/1429107785_678092-sign-add-128.png</normaloff>:/icons/icons/1429107785_678092-sign-add-128.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editFoodDetails">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Edit Food &amp; Beverages Detaiils</string>
       </property>
       <property name="icon">
        <iconset resource="bill_master_resources.qrc">
         <normaloff>:/icons/icons/1428360743_file_edit.png</normaloff>:/icons/icons/1428360743_file_edit.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="deleteFoods">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Delete Foods and Beverages</string>
       </property>
       <property name="icon">
        <iconset resource="bill_master_resources.qrc">
         <normaloff>:/icons/icons/1429107797_678069-sign-error-128.png</normaloff>:/icons/icons/1429107797_678069-sign-error-128.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Food List</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLineEdit" name="searchBox">
            <property name="placeholderText">
             <string>Search Text</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="searchBtn">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="reload">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="bill_master_resources.qrc">
              <normaloff>:/icons/icons/1434209948_Synchronize.png</normaloff>:/icons/icons/1434209948_Synchronize.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="foodListTable">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <column>
           <property name="text">
            <string>HSN Code</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Category</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>SGST (%)</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>CGST (%)</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Price</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Food Orders</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Order List</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="0">
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="2">
                <widget class="QPushButton" name="removeOrder">
                 <property name="text">
                  <string>Remove Order</string>
                 </property>
                 <property name="icon">
                  <iconset resource="bill_master_resources.qrc">
                   <normaloff>:/icons/icons/1429107797_678069-sign-error-128.png</normaloff>:/icons/icons/1429107797_678069-sign-error-128.png</iconset>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="finalizeFoods">
                 <property name="text">
                  <string>Finalize Foods</string>
                 </property>
                 <property name="icon">
                  <iconset resource="bill_master_resources.qrc">
                   <normaloff>:/icons/icons/1440196879_tick_blue.png</normaloff>:/icons/icons/1440196879_tick_blue.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTableWidget" name="orderTable">
               <property name="minimumSize">
                <size>
                 <width>400</width>
                 <height>0</height>
                </size>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <column>
                <property name="text">
                 <string>Item</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Qty</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Time</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Price</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Booking Details</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Month</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="6">
                <widget class="QPushButton" name="addOrder">
                 <property name="text">
                  <string>Add Order</string>
                 </property>
                 <property name="icon">
                  <iconset resource="bill_master_resources.qrc">
                   <normaloff>:/icons/icons/fff-add.ico</normaloff>:/icons/icons/fff-add.ico</iconset>
                 </property>
                </widget>
               </item>
               <item row="0" column="5">
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Year</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QComboBox" name="yearList"/>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="monthList"/>
               </item>
               <item row="0" column="4">
                <widget class="QPushButton" name="pushButton">
                 <property name="text">
                  <string>Filter</string>
                 </property>
                 <property name="icon">
                  <iconset resource="bill_master_resources.qrc">
                   <normaloff>:/icons/icons/1434208652_filter.png</normaloff>:/icons/icons/1434208652_filter.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTableWidget" name="bookingTable">
               <property name="minimumSize">
                <size>
                 <width>400</width>
                 <height>0</height>
                </size>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>Booking Serial</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Customer Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Room No</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Booking From</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Booking To</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="bill_master_resources.qrc"/>
 </resources>
 <connections/>
</ui>
